src/interpret.zig

```diff
@@ -358,7 +370,7 @@ pub const DtVal = union(enum) {
 
     pub fn intoQuote(self: DtVal, state: *DtMachine) !Quote {
         return switch (self) {
-            .quote => |q| _deepClone(q, state),
```

Comment:
TODO: Must add tests to ensure this is actually safe. Hopefully simple tests of `def` and `dup` and others will sort out exactly where things should be cloned.